### DISCUSSION TITLE
Syntax mention!

### DISCUSSION THREAD

**Discussion by SrGeneroso** [pocketbase/pocketbase#5024] [Show and tell]
Pocketbase is in the syntax podcast! 
https://www.youtube.com/watch?v=Eg38JbgbttA

**Comment by SrGeneroso** [comment]
Scott loves it! A lot of deserved praises and a quick slick demo. Be prepared to a ton of newbies coming.

**Comment by TobiasNickel** [comment]
I watched the video, it is good. I wish there is more content about pocketbase.

**Reply by SrGeneroso** [reply]
being still in beta I understand content creators don't want to, there is a risk to share soon to be outdated data
